www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
 ---
 sidebar_links:
-- title: Getting Started
+- title:
   links:
-  - title: Overview
-    link: "/docs/overview/"
-  - title: Download and install Habitat
-    link: "/tutorials/download/"
-  - title: Tutorials
-    link: "/tutorials/"
-  - title: Concepts <span>&plus;</span>
-    link: "/docs/concepts-overview/"
+  - title: Docs Home
+    link: "/docs"
+  - title: Using Habitat <span>&plus;</span>
```

Comment:
I'm worried that having several large topics will mess up site search and SEO for search terms.

Reply:
The large topics are much more in line with typical developer documentation. Developers like to be able to ctrl-f for content instead of needing to find it under a subcategory. Let's adjust search as needed.

Reply:
@davidwrede what kind of issues are you anticipating? We should weigh them against the nice to have of being able to ctrl-f an entire document (which is really appreciated on a devs part)

Reply:
@reset @tashimi For content that has a logical grouping and has an expected experience (e.g. going to a REST API ref page that lists all of the endpoints), it definitely makes a lot of sense to have one big uber page.

I'm worried that by having one large topic that covers many different areas (such as under the "Using Habitat") topic, content that is far down on the page (like "Update Strategies") will not show up in correctly in search (meaning the topic may be returned further down on the list or not be in the list at all). Content that tends to show up well in search results have the same keywords in the filename, title, body of text, metadata description, etc. indicating that this topic is indeed related to the keywords that you're searching on.

Having said that, we can go ahead with getting this merged and I'll keep a watch out to see how this change will affect discoverability.